lib/plaid/models.rb

```diff
@@ -7,10 +7,15 @@ class BaseModel < Hashie::Dash
       include Hashie::Extensions::Dash::IndifferentAccess
       include Hashie::Extensions::Dash::Coercion
 
+      @@ignored_properties = []
```

Comment:
@be9 introduced this because there is a `institution.input_spec` prop that we want to ignore when `relaxed_models = false` but we don't want to support on the API